vigor/static/css/style.css

```diff
@@ -0,0 +1,73 @@
+@import "../bower_components/material-typography/css/typography.css";
```

Comment:
You know what? I actually have material called in the actual `style.css` file. Maybe that's what's happening, too